reference/test/reorder/mc64_kernels.cpp

```diff
@@ -373,6 +379,8 @@ TYPED_TEST(Mc64, CreatesCorrectPermutationAndScalingLargeExampleProduct)
     using value_type = typename TestFixture::value_type;
     using matrix_type = typename TestFixture::matrix_type;
     using perm_type = typename TestFixture::perm_type;
+    // this example can not be finished in half precision
```

Comment:
I guess that's because the matrix contains values outside the value range of half? Maybe add that to the comment.